packages/did-core-test-server/suites/did-consumption/default.js

```diff
@@ -1,53 +1,10 @@
 module.exports = {
-  conformingConsumers: [{
```

Comment:
Removal of this and replacement with actual DID Methods is correct -- old consumer tests were using old consumer data (and not using implementations).